daps_utils/docker_utils.py

```diff
@@ -150,7 +150,9 @@ def build_flow_image(pkg, flow_path, rebuild_base,
                                    f'"{base_tag}".')
     # Now build the base image
     build_image(pkg=pkg, tag=base_tag, dockerfile=base_dockerfile,
-                rebuild=rebuild_base)
+                rebuild=rebuild_base,
+                rm=True #  Remove intermediate containers
```

Comment:
(doing this now, will update this comment)

Reply:
Can confirm that doesn't seem to make a difference

Reply:
Thanks Joel for confirming. I am resolving this comment.
I believe this can be revisited if it drastically affects performance during local development.